python/numpy/timings/black_scholes_config.py

```diff
@@ -0,0 +1,21 @@
+#
+# Configuration for the Black Scholes workload.
```

Comment:
Same here - These things should probably go in their own repo under weld-project instead of being here.